test/test-coordinator.js

```diff
@@ -245,6 +247,19 @@ TestCoordinator.prototype.callEndpoint = function callEndpoint(endpoint, body, c
 TestCoordinator.prototype.getMembership = function getMembership() {
     return this.getFakeNodes().map(function(node) {
         return node.toMemberInfo();
+    }).concat(this.extraMembers);
+};
+
+TestCoordinator.prototype.extraMember = function(address, status, incarnationNumber) {
```

Comment:
I want to distinguish it from the fake nodes. Which are not called members, but I named it like this to make it more clear that we are adding extra information to the membership list instead of starting an other full member of the ring.